zebra-state/src/service/finalized_state.rs

```diff
@@ -599,6 +599,15 @@ impl FinalizedState {
             .zs_get(value_pool_cf, &())
             .unwrap_or_else(ValueBalance::zero)
     }
+
+    /// Allow to set up a fake value pool in the database for testing purposes.
+    #[cfg(test)]
```

Comment:
We might want to use this method from `zebra-consensus` at some point:
```suggestion
    #[cfg(any(test, feature = "proptest-impl"))]
```

Reply:
I had it like this before but removed because of:

```
warning: associated function is never used: `set_current_value_pool`
   --> zebra-state/src/service/finalized_state.rs:605:12
    |
605 |     pub fn set_current_value_pool(&self, fake_value_pool: ValueBalance<NonNegative>) {
    |            ^^^^^^^^^^^^^^^^^^^^^^
    |
    = note: `#[warn(dead_code)]` on by default

```

We need something else to do it this way. not sure what yet.

Reply:
It's ok to allow dead code, or to just leave it and fix it later.
```suggestion
    #[cfg(any(test, feature = "proptest-impl"))]
    #[allow(dead_code)]
```

Reply:
Ok, added dead code at https://github.com/ZcashFoundation/zebra/pull/2652/commits/f923006612b10bcc43195755bd9abb05ada48a9e